tool/resources/org/antlr/v4/tool/templates/codegen/Java/Java.stg

```diff
@@ -210,14 +215,25 @@ public class <file.grammarName>BaseVisitor\<T> extends AbstractParseTreeVisitor\
  * \<p>The default implementation returns the result of calling
  * {@link #visitChildren\} on {@code ctx\}.\</p>
  */
-@Override public T visit<lname; format="cap">(<file.parserName>.<lname; format="cap">Context ctx) { return visitChildren(ctx); \}}; separator="\n">
+@Override public T visit<lname; format="cap">(<notNull(file)><file.parserName>.<lname; format="cap">Context ctx) { return visitChildren(ctx); \}}; separator="\n">
 }
 >>
 
 fileHeader(grammarFileName, ANTLRVersion) ::= <<
 // Generated from <grammarFileName; format="java-escape"> by ANTLR <ANTLRVersion>
 >>
 
+importNullAnnotations(elem) ::= <<
+<if(elem.nullAnnotationsPackage)>
```

Comment:
seems like we should just always generate a ref to the appropriate package rather than an option.